### PR TITLE
Added sample code to make the WP CFM work in multidev

### PR DESCRIPTION
**Pull request by carl-alberto**
Closes: #5865

## Summary

**[Create a WordPress MU-Plugin for Actions and Filters](https://pantheon.io/docs/mu-plugin#wp-cfm-compatibility)** - Added sample code to enable WP-CFM support for Multidev environments. Referenced and crosslinked in [WordPress Configuration Management (WP-CFM)](https://pantheon.io/docs/wp-cfm#will-wp-cfm-work-with-multidev)

## Post Launch

- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-5964](https://pr-5964--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `56.99999999999999` | `56.00000000000001` | `-1` |
| Best Practice Score | `93` | `93` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `93` | `93` | `0` |

Lighthouse Audit Reports:

- [`pr-5964` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/23395/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/23395/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-5964--pantheon-docs.my.pantheonfrontend.website/docs)

File: source/content/wp-cfm.md [hunk 1]
```diff
@@ -130,12 +143,19 @@ Deploy the `.json` file from Dev to Test.
 You can review values on the [All Settings Screen](https://codex.wordpress.org/Option_Reference#All_Settings_Screen) (`/wp-admin/options.php`).
 
 ### How can I extend WP-CFM to track more tables?
+
 If you want to track configurations in more tables, you must do so using the `wpcfm_configuration_items` hook. For details, see [WP-CFM documentation](https://forumone.github.io/wp-cfm/).
 
+### Will WP-CFM work with Multidev?
+
+Yes. For the Multidev to appear as a config option, you will need hook into the plugin's [`wpcfm_current_env`](https://github.com/forumone/wp-cfm/wiki/Filters-Reference#wpcfm_current_env) and [`wpcfm_multi_env`](https://github.com/forumone/wp-cfm/wiki/Filters-Reference#wpcfm_multi_env) fucntions, in a Must Use Plugin like the examples here setup [MU Plugin Bar](/mu-plugin/#example-code-snippets).
```
Comment (alexfornuto) [inline source/content/wp-cfm.md:151]: @carl-alberto I've made some updates here, but it needs further clarification. What do you mean by "Plugin Bar"?

Would you agree with my technical edits and references to the functions?

**Review by carl-alberto — COMMENTED** [review]
Sorry for the confusion @alexfornuto, **MU Plugin Bar** should be just plain **MU Plugin**

So basically the thought is hook up with `wpcfm_current_env` and `wpcfm_multi_env` set it up as a Must Use plugin using the example code that I have in this PR https://github.com/pantheon-systems/documentation/pull/5963/commits/1efe75dad0d95fce6f737167e14f09c726bfba57

**Review by alexfornuto — APPROVED** [review]
Thanks @carl-alberto!

**Pull request merged**